### PR TITLE
Comments out 'herald' breakdown

### PR DESCRIPTION
**Pull request by bearrrrrrrr**
## About The Pull Request

Does what it says on the tin. Comments out /datum/breakdown/common/herald and /datum/breakdown/common/herald/update(). I asked about this on discord and nobody really objected to it.

To clarify, this is the breakdown that makes you speak in chat with a breakdown message (not even unique ones), every few seconds.

## Why It's Good For The Game

This is a breakdown that, IC'ly, didn't add a while lot that other breakdowns didn't; a huge lapse in sanity. There are other breakdowns that simulate having a nervous attack/whatever you would call this. These do things with ingame mechanics, such as making you obsessed over a person or item, or making everyone look like _weird spooky symbols,_ or having a hysteric breakdown. The only mechanic this provides is you spamming chat rapidly. It also entirely demystifies the insanity messages, making rare once-every-ten-minute occurences into every-ten-second ones. 

It's hard to RP, makes it hard to RP, and removes a genuine allure of mystery.

## Changelog
```changelog
del: Commented out herald breakdown.
```

### CODE REVIEW

**Review by Michiyamenotehifunana — APPROVED** [review]
Codewise, looks good. Could probably use an emojicracy before merging though.

**Comment by Jamini** [comment]
I'm going to hold off on this until an Emojicracy has went through

**Comment by Jamini** [comment]
Resolved in #8

**Pull request closed without merging**
